homeassistant/components/google_assistant_sdk/__init__.py

```diff
@@ -57,6 +60,8 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     hass.data.setdefault(DOMAIN, {})[entry.entry_id] = session
 
     await async_setup_service(hass)
+    agent = GoogleAssistantConversationAgent(hass, entry)
+    conversation.async_set_agent(hass, agent)
```

Comment:
Right now Home Assistant can only handle a single conversation agent. The default one uses the new intent system (to be released in February). So can we make setting Google as the conversation agent configurable via the options flow? 

We will probably tweak it later to give the conversation agent a UI with a picker.

Reply:
I added an option but the strings I added (dialog description and new field) are missing from the UI 
<img width="311" alt="missing text" src="https://user-images.githubusercontent.com/9987465/211397409-e8ba275b-e285-4390-a7da-6f868aa73ff8.png">